Makefile

```diff
@@ -6,6 +6,8 @@ docker/cloud/start:
 	docker run -it -d --name go-nitro-cloud \
 	  -p 3005:3005 -p 4005:4005 -p 5005:5005 \
 		-e NITRO_CONFIG_PATH="./nitro_config/iris.toml" \
+		-v ./tls:/app/tls \
+		-v ./data:/app/data \
```

Comment:
Did you mean to include these changes on this PR?

Reply:
That was intentional. I had to add those volumes in order to run the tests mentioned in this PR's description. The `deploy-cloud-nodes` workflow uses its own custom `docker run` command so this `make` command is only used locally and does not interfere with anything else at the moment